packages/kbn-test/src/functional_tests/run_tests/run_tests.ts

```diff
@@ -67,7 +68,13 @@ export async function runTests(log: ToolingLog, options: RunTestsOptions) {
         log.write(`--- [${progress}] Running ${Path.relative(REPO_ROOT, path)}`);
       }
 
-      const config = await readConfigFile(log, options.esVersion, path, settingOverrides);
+      const config = await readConfigFile(
+        log,
+        options.esVersion,
+        path,
+        settingOverrides,
+        applyFipsOverrides
+      );
```

Comment:
I'm not sure, but since `applyFipsOverrides` is optional I wonder if we better not pass it by default, but only if env variable is set.

Reply:
++ Ill change it